proto/accessgraph/v1alpha/query.proto

```diff
@@ -29,11 +29,13 @@ service AccessGraphService {
   // GetFile gets a static UI file from the access graph container.
   rpc GetFile(GetFileRequest) returns (GetFileResponse);
 
-  // SendEvent sends an event to the access graph service.
-  rpc SendEvent(SendEventRequest) returns (SendEventResponse);
-
-  // SendResource sends a resource to the access graph service.
-  rpc SendResource(SendResourceRequest) returns (SendResourceResponse);
+  // EventsStream is a stream of commands to the access graph service.
+  // Teleport Auth server creates a stream to the access graph service
+  // and pushes all resources and following events to it.
+  // This stream is used to sync the access graph with the Teleport database state.
+  // Once Teleport finishes syncing the current state, it sends a sync command
+  // to the access graph service and resumes sending events.
+  rpc EventsStream(stream EventsStreamRequest) returns (EventsStreamResponse);
```

Comment:
An unidirectional stream leaves you with little recourse to reply with errors to specific messages. This might be what you want, I just wanted to raise to attention to it.

Reply:
This is what we need for now. If there is any error on the server side we need to stop the streaming and figure out what happen. We have an idea on V2 to address that, but for now we need this version.